dev-client/src/App.tsx

```diff
@@ -76,8 +76,8 @@ function App(): React.JSX.Element {
     <GestureHandlerRootView style={style}>
       <Provider store={store}>
         <NativeBaseProvider theme={theme}>
-          <BottomSheetModalProvider>
-            <Portal.Host>
+          <Portal.Host>
+            <BottomSheetModalProvider>
```

Comment:
comment: @tm-ruxandra i got an app crash due to `BottomSheetModalProvider` being in the wrong place, this change is just a quick fix to be able to validate this fix but i wanted to check if you ever did any more work on this or if i'm good to do the fix we talked about (moving it into screen scaffold, below the header height provider) without stepping on your toes

Reply:
@shrouxm Totally fine if you need to make this change here, it won't conflict with any of my ongoing work.